hardware/setup/create_framework.tcl

```diff
@@ -279,8 +279,8 @@ if { ($fpga_card == "N250SP") && ($capi_bsp_dir != "not defined") } {
 puts "                        importing XDCs"
 
 # Board Support XDC
-if { ($fpga_card == "N250SP") && ($capi_bsp_dir != "not defined") } {
-  puts "                        importing N250SP Board support XDCs"
+if { (($fpga_card == "N250SP") || ($fpga_card == "RCXVUP")) && ($capi_bsp_dir != "not defined") } {
+  puts "                      importing specific Board support XDCs"
```

Comment:
The format of the message is changed compared to master (removing the two spaces in front of `importing` - maybe re-introducing a previous formatting error).
Thomas won't like that the message is no longer aligned to the other messages in this level ... ;-)

Reply:
good catch . corrected